SampleWebApp/App_Start/Startup.Auth.cs

```diff
@@ -37,6 +38,7 @@ public ConditionalMiddlewareInvoker(OwinMiddleware next)
         public void Configuration(IAppBuilder app)
         {
             app.Use(typeof(ConditionalMiddlewareInvoker));
+            app.UseStageMarker(PipelineStage.Authenticate);
```

Comment:
Is the presence of this line this a workaround for the hack with the ConditionalMiddlewareInvoker? Is it needed for the proof of concept app to work? If not, I'd prefer to have it removed. If it is needed, a comment would be appropriate (I'll add a comment on what the ConditionalMiddlewareInvoker is too, as it is definitely non-standard).